examples/js/postprocessing/EffectComposer.js

```diff
@@ -58,7 +58,7 @@ THREE.EffectComposer = function ( renderer, renderTarget ) {
 
 	this.copyPass = new THREE.ShaderPass( THREE.CopyShader );
 
-	this._previousFrameTime = Date.now();
+	this.clock = new THREE.Clock();
```

Comment:
Since it's a public property now, it would be good to add it to the TS declaration file.